lib/auth/exec.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const { execFile } = require('child_process'),
+      proc         = require('process'),
```

Comment:
I would avoid requiring `process` and maybe rename the eponym variable to `exec` and just use `process.env`.

Reply:
Done ( renamed it environment  )

Reply:
I meant just use `process.env` inline everywhere as `process` is global, but I realise my first comment was not crystal clear 😉.

Reply:
Haha much clearer :+1: